tests/conftest.py

```diff
@@ -2,6 +2,7 @@
 import pytest
 import re
 from . import cmake, run
+from .cmake import CMake
```

Comment:
low|nit|whatever: I'm fairly sure pep8 style says something akin to "import modules, not things within them"